kuma/wiki/tests/test_templates.py

```diff
@@ -245,7 +246,7 @@ def test_revision_view(self):
         """Load the revision view page and verify the title and content."""
         d = _create_document()
         r = d.current_revision
-        r.created = datetime(2011, 1, 1)
+        r.created = datetime(2011, 1, 1, tzinfo=timezone.utc)
```

Comment:
Who the hell is this guy and why did he use utc for god's sake? For real, this should use the default timezone I think.